easybuild/easyconfigs/c/Cython/Cython-0.26-GCCcore-6.4.0-Python-2.7.13-bare.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.26'
+pyver = '2.7.13'
+pyshortver = '2.7'
+pysubver = '-bare'
+versionsuffix = '-Python-%s%s' % (pyver, pysubver)
```

Comment:
What you are basically doing is dancing around the fact that you're not including Python as a runtime dependency, while it actually *is* a runtime dependency? To me (currently), that's a sign that this may be the wrong approach.